CONTRIBUTING.md

```diff
@@ -79,6 +79,7 @@ optimize:
 When Kibana runs in development mode it will automatically use bundled SSL certificates. These certificates won't be trusted by your OS by default which will likely cause your browser to complain about the cert. You can deal with this in a few ways:
 
   1. Supply your own cert using the `config/kibana.dev.yml` file.
+    - You can point at the development certificate found in `test/dev_certs` 
```

Comment:
Actually @jbudz was right. What confused me was the numbered bullet list. I thought I had to follow every point (supply my own cert, mark it trusted, etc). That tripped me up. I'll make it into an unordered list.